### PR TITLE
Api 1.9.0

### PR DESCRIPTION
**Pull request by Guy-Moses**
Advance API version to v1.9.0
(Adds launchURL API)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This page contains information about changes to the PowerBI Visual Tools (pbiviz).
 
+## pbiviz v1.9.0
+* Added launchUrl API call support
 ## pbiviz v1.8.1
```
Comment (AviSander) [inline Changelog.md]: CR LF

File: spec/e2e/pbivizNewSpec.js [hunk 2]
```diff
@@ -182,7 +182,7 @@ describe("E2E - pbiviz new", () => {
                     "fontSize": "21"
                 }
             };
-
+            debugger; // jshint ignore:line
```
Comment (AviSander) [inline spec/e2e/pbivizNewSpec.js]: remove

File: templates/visuals/.api/v1.9.0/schema.capabilities.json [hunk 3]
```diff
@@ -0,0 +1,1131 @@
+{
```
Comment (AviSander) [inline templates/visuals/.api/v1.9.0/schema.capabilities.json]: verify merged with Master

File: spec/e2e/pbivizNewSpec.js (hunk 2 above)
Comment (Guy-Moses) [reply spec/e2e/pbivizNewSpec.js]: Done

File: Changelog.md (hunk 1 above)
Comment (Guy-Moses) [reply Changelog.md]: Done

File: templates/visuals/.api/v1.9.0/schema.capabilities.json (hunk 3 above)
Comment (Guy-Moses) [reply templates/visuals/.api/v1.9.0/schema.capabilities.json]: Verified - nothing to change...

**Pull request merged**